packages/@aws-cdk/aws-codepipeline-actions/README.md

```diff
@@ -997,3 +997,31 @@ pipeline.addStage({
 
 See [the AWS documentation](https://docs.aws.amazon.com/codepipeline/latest/userguide/action-reference-StepFunctions.html)
 for information on Action structure reference.
+
+### AWS Stack Sets
```

Comment:
Let's move this to the CloudFormation section of the ReadMe: https://github.com/aws/aws-cdk/blob/e7f09cb5159fbfad3a5c5fcbbc1fa8eab3d711dd/packages/@aws-cdk/aws-codepipeline-actions/README.md#aws-cloudformation

Reply:
does the updated one work? I got rid of the example since most things under cfn don't have examples.